tests/test_proxy.py

```diff
@@ -44,10 +44,10 @@ def test_recv_reads_from_process_stdout_returning_bytes(
         select.return_value = [stdout], None, None
         fileno = stdout.fileno.return_value
         # Intentionally returning <5 at a time sometimes
-        os_read.side_effect = [b"was", b"te", b"of ti", b"me"]
+        os_read.side_effect = [b"123", b"45", b"of to", b"me"]
```

Comment:
> The willingness of codespell to rewrite string literals is concerning to me. I would not be comfortable with allowing it free hand to rewrite code---e.g., in a pre-commit hook---with putting it in CI.

these changes here were done manually by me in 5efa354ce9fff1217d22e52493d1442866821a7b .
The only automated changes done by **explicit manual run** with subsequent analysis of diff were done via `codespell -w` in 908e7c75f53025a3064e28215da62ffcfdea6daa . 
CI workflow would only **alert** on present typos without doing any automagic fixes. I have not added/proposed pre-commit setup with it.

 Re this specific change -- since content of those strings doesn't really matter, I just replaced with more obvious to me `12345` as the target. Probably should have extended to all characters, so what about

```suggestion
        os_read.side_effect = [b"123", b"45", b"67 89", b"10"]
```
?

Reply:
> Did you notice that these strings are the phrase 'waste of time' broken up? 

yes, eventually I did but not at once

> By changing these to numbers, as with the commit below, you've removed some of the developers' commentary from the test.

hm, without looking anywhere else in the code, not clear to me how this `.side_effect` is the commentary.  Might be more obvious for silly ones like me to just leave a comment ;) (said hypocritical me who does like generally to use descriptive names like `"non-existing"` for non-existing files etc)

Reply:
I think it's more an expression of the dev's frustration (not sure if it's bitprophet or not, would have to git blame it) at needing to write the test, as opposed to code comments meant to explain what the test is doing.